phys2denoise.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python3
+
+"""
+Phys2denoise is a python3 library meant to prepare physiological regressors for fMRI denoising.
+
+The project is under development.
+
+Copyright 2020, The physiopy community.
+Please scroll to bottom to read full license.
+
+"""
+
+import datetime
+import logging
+import os
+import sys
+from copy import deepcopy
+from shutil import copy as cp
+
+import numpy as np
+
+from phys2denoise import utils, viz, _version
+from phys2denoise.cli.run import _get_parser
+# from phys2denoise.metrics import cardiac, chest_belt, retroicor
+
+from . import __version__
+from .due import due, Doi
+
+LGR = logging.getLogger(__name__)
+
+
+def print_json(outfile, samp_freq, time_offset, ch_name):
+    """
+    Print the json required by BIDS format.
```

Comment:
Any sidecar files generated by the tool should be BIDS _Derivatives_-compatible, specifically. The requirements/recommendations are somewhat different compared to raw data sidecars.